### ISSUE TITLE
Getting "Error: ENOENT: no such file or directory" when trying to run amplify init.

### ISSUE THREAD

**Issue by AmitTzah** [aws-amplify/amplify-cli#9244]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

curl -sL https://aws-amplify.github.io/amplify-cli/install-win -o install.cmd && install.cmd

### If applicable, what version of Node.js are you using?

v16.13.1

### Amplify CLI Version

7.6.2

### What operating system are you using?

Windows 10

### Amplify Categories

Not applicable

### Amplify Commands

init

### Describe the bug

After creating a new flutter project, running 'amplify init' results in the error:

Error: ENOENT: no such file or directory, open 'C:\Users\Username\AppData\Roaming\SPB_Data\.aws\config'

at Object.openSync (fs.js:462:3)

at Object.openSync (pkg/prelude/bootstrap.js:739:32)

at Object.readFileSync (fs.js:364:35)

at Object.readFileSync (pkg/prelude/bootstrap.js:1025:36)

at Object.readFileSync (C:\snapshot\node_modules\aws-sdk\lib\util.js:95:26)

at IniLoader.parseFile (C:\snapshot\node_modules\aws-sdk\lib\shared-ini\ini-loader.js:6:47)

at IniLoader.loadFrom (C:\snapshot\node_modules\aws-sdk\lib\shared-ini\ini-loader.js:56:30)

at getRegion (C:\snapshot\node_modules\aws-sdk\lib\node_loader.js:116:32)

at Config.region (C:\snapshot\node_modules\aws-sdk\lib\node_loader.js:171:18)

at Config.set (C:\snapshot\node_modules\aws-sdk\lib\config.js:524:39)

at Config.<anonymous> (C:\snapshot\node_modules\aws-sdk\lib\config.js:359:12)

at Config.each (C:\snapshot\node_modules\aws-sdk\lib\util.js:512:32)

at new Config (C:\snapshot\node_modules\aws-sdk\lib\config.js:358:19)

at Object.loadServiceClass (C:\snapshot\node_modules\aws-sdk\lib\service.js:108:16)

at Object.Service [as constructor] (C:\snapshot\node_modules\aws-sdk\lib\service.js:47:29)

at new <anonymous> (C:\snapshot\node_modules\aws-sdk\lib\util.js:629:24)

at getConfiguredAmplifyClient (C:\snapshot\node_modules\amplify-provider-awscloudformation\lib\aws-utils\aws-amplify.js:42:14)

at processTicksAndRejections (internal/process/task_queues.js:97:5) {

errno: -4058,

syscall: 'open',

code: 'ENOENT',

### Expected behavior

Initialization of a new project correctly.

### Reproduction steps

 1.  install Amplify CLI via:

curl -sL https://aws-amplify.github.io/amplify-cli/install-win -o install.cmd && install.cmd

2. Run  'amplify configure' and create an IAM user.
3. Create a new flutter project inside VScode.
4. Run 'amplify init' from the root directory of the new flutter project.

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

### Additional information

The folder .aws specified in the error message is missing from C:\Users\Username\AppData\Roaming\SPB_Data\

**Comment by akshbhu** [comment]
Hi @AmitTzah 

Can you check if the config file is present inside .aws folder? We use the folder to fetch credentials for amplify IAM user to initialize amplify project.

**Comment by AmitTzah** [comment]
Hi @akshbhu 

Thank you for responding.

The folder .aws is missing from  C:\Users\Username\AppData\Roaming\SPB_Data.

Searching for  C:\Users\Username\AppData\Roaming\SPB_Data\.aws in the file explorer results in the messeage "windows is unable to find C:\Users\Username\AppData\Roaming\SPB_Data\.aws

**Comment by akshbhu** [comment]
Can you try running amplify configure again and check   if the  config files are getting generated at the location?

**Comment by akshbhu** [comment]
Can you try running amplify configure again and check   if the  config files are getting generated at the location?

**Comment by AmitTzah** [comment]
Yes, I've created three different IAM profiles using the amplify configure command, and the .aws folder isn't being created.

**Comment by akshbhu** [comment]
I will try to reproduce this on my end , for the workaround can you try our npm approach?
https://docs.amplify.aws/cli/start/install/

**Comment by AmitTzah** [comment]
Yes, I will reinstall Amplify CLI with NPM and will report back. 
Also, I've noticed that the folder SPB_Data is "read-only" (right-click on folder>properties>attributes>Read-only(Only applies to files in folder).
And I can't seem to uncheck it.
Could it prevent Amplify from generating the needed files?

**Comment by akshbhu** [comment]
Yes, this can be a reason , since we write the aws config files in the directory.
Can you also try with administrator access ?

**Comment by AmitTzah** [comment]
for some reason SPB_Data folder keeps reverting to read only on, when I try to uncheck it.

Do you run Amplify configure with administrator access? How do I do that?

**Comment by akshbhu** [comment]
Try running terminal with administrator  accces and  again run this command

**Comment by AmitTzah** [comment]
I ran the terminal with admin access and created yet another profile via Amplify configure. Still getting the same error and ,aws folder is not being generated.

Installing Amplify CLI via npm produces the same problem.

**Comment by AmitTzah** [comment]
Update: I solved the problem.

I lucked out in noticing that .aws folder was being generated in C:\Users\Username so I simply copied it to  C:\Users\Username\AppData\Roaming\SPB_Data where it'd been missing.

The only problem now is when running additional Amplify configure commands, .aws is being updated only in C:\Users\Username , so I have to remember to copy paste to SPB_Data  after every new profile is created.
